pallets/loans/src/math.rs

```diff
@@ -31,43 +30,36 @@ pub fn calculate_accumulated_rate<Rate: FixedPointNumber>(
 		.and_then(|v| v.checked_mul(&current_accumulated_rate))
 }
 
-/// converts a fixed point from A precision to B precision
-/// we don't convert from un-signed to signed or vice-verse
-pub fn convert<A: FixedPointNumber, B: FixedPointNumber>(a: A) -> Option<B> {
-	if A::SIGNED != B::SIGNED {
-		return None;
-	}
-
-	B::checked_from_rational(a.into_inner(), A::accuracy())
-}
-
 /// calculates the debt using debt=principal_debt * cumulative_rate
-pub fn debt<Amount: FixedPointNumber, Rate: FixedPointNumber>(
-	principal_debt: Amount,
+pub fn debt<Balance: FixedPointOperand, Rate: FixedPointNumber>(
+	normalized_debt: Balance,
 	accumulated_rate: Rate,
-) -> Option<Amount> {
-	convert::<Rate, Amount>(accumulated_rate).and_then(|rate| principal_debt.checked_mul(&rate))
+) -> Option<Balance> {
+	accumulated_rate.checked_mul_int(normalized_debt)
 }
 
-pub enum Adjustment<Amount: FixedPointNumber> {
-	Inc(Amount),
-	Dec(Amount),
+pub enum Adjustment<Balance> {
+	Inc(Balance),
+	Dec(Balance),
 }
```

Comment:
I would leave a comment here explaining that it's meant to represent how much some other value should be incremented or decremented by. When I first read this, I thought this was meant to be Like `Inc(x) == x+1`.

Reply:
Done!